Control coverage/Feature/VirtualMachineScaleSet.md

```diff
@@ -739,4 +740,77 @@ Using this feature ensures that sensitive data is stored encrypted at rest. This
   <br />
  <br />
 
+___
+
+## Azure_VirtualMachineScaleSet_AuthN_Enable_AAD_Auth_Linux_Trial
```

Comment:
is this control available for external users?

Reply:
Trial is removed and this control is available for external users